server.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+import socket
+
+
+def response_ok():
+    header = 'HTTP/1.1 200 OK\nContent-Type: text/html'
+    body = '<html><body>200 OK</body></html>'
+    return header + body
+
+
+def response_error():
+    response = 'HTTP/1.1 500 Internal Server Error\n'
+    return response
```

Comment:
In both of these cases we are missing the required `CRLF` separator that must end each line.  I think you'll be fixing that in later stages.